metrics/sinks/stackdriver/stackdriver.go

```diff
@@ -460,6 +460,20 @@ func (sink *StackdriverSink) LegacyTranslateLabeledMetric(timestamp time.Time, l
 			"device_name": metric.Labels[core.LabelResourceID.Key],
 		}
 		return ts
+	case core.MetricFilesystemInodesFree.MetricDescriptor.Name:
```

Comment:
I think the same change has to be made to TranslateLabeledMetrics method, @kawych is it correct?

Reply:
Yes, if we add a new metric, we should have the same metric for new resource model. Can you provide more context behind adding this metric? We should decide what metric do we need and what monitored resource to use.

Also, this metric needs to be defined in Stackdriver first, doesn't it?

Reply:
CLA is signed, not sure if the bot will pick it up automatically or not. :)

The reason for this FR is discussed in b/73268296. Since cadvisor's summary api provides inodes metrics already, I see no reason why we shouldn't sink them to SD.

Reply:
Thanks for the context, I agree we should add the metric. I've asked folks from Stackdriver to take a look, when the Stackdriver change is in place we can merge this PR.